source/upgrade.txt

```diff
@@ -85,21 +87,18 @@ Version 5.0 Breaking Changes
   - ``BsonProperty``
   - ``BsonRepresentation``
 
-- You must pass a timeout duration, which is the first parameter, to the
-  following ``SocketSettings`` builder methods as a ``long`` type:
-
-  - ``connectTimeout()``
-  - ``readTimeout()``
-
-  In earlier versions, this parameter is of type ``int`` for both methods. To view an example
-  that shows how to call ``SocketSettings`` methods, see the :ref:`SocketSettings Example
-  <java-socketsettings-example>` in the Specify MongoClient Settings
-  guide.
-
-  This change breaks binary compatibility (requires recompiling) but does not
-  require code changes.
-
-- This driver version removes the ``Filters.eqFull()`` method, released
+- Changes the data type of the timeout duration parameter in the
+  ``SocketSettings.Builder.connectTimeout()`` and
+  ``SocketSettings.Builder.readTimeout()`` methods. The data type of this
+  parameter is now ``long`` instead of ``int``.
+  
+  In earlier versions, this parameter is of type ``int`` for both methods. This
+  change breaks binary compatibility and requires recompiling, but does not
+  require code changes. To view an example that shows how to call
+  ``SocketSettings`` methods, see the :ref:`SocketSettings Example
+  <java-socketsettings-example>` in the Specify MongoClient Settings guide.
```

Comment:
While this appears at first glance to be a major change, it's actually a minor rewrite to accommodate the new phrasing.